addons/knowledge/static/src/scss/knowledge_views.scss

```diff
@@ -46,6 +46,10 @@
         min-height: 2rem;
     }
 
+    a.o_article_emoji:hover {
+        background-color: rgba(#000, .1);
+    }
```

Comment:
Is this still part of the sidebar or not?

(It's included in a "Sidebar" section)